app/controllers/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
```

Comment:
...here.

if it's a class inside a class:
```
class Users
  class OmniauthCallbacksController
    ...
  end
end
```

If it's a module
```
module Users
  class OmniauthCallbacksController
    ...
  end
end
```

But never module Module::ClassName or class Class::ClassName